IFC4x3/Properties/l/LogNo_2H$tb9GtfCP9DRUp61IODv/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LogNo_2H$tb9GtfCP9DRUp61IODv" Name="LogNo" UniqueId="91ff7949-437a-4c64-935b-7b3181498379" PrimaryDataType="IfcInteger" />
```

Comment:
I would prefer a more descriptive name but I guess it is fine if the stakeholders say so.

Reply:
Changed to LogNumber in [ebc851b](https://github.com/bSI-InfraRoom/IFC-Specification/pull/589/commits/ebc851b49d47f6426cb687367b124ed31c9053cf)